### ISSUE TITLE
Focus navigation is trapped from DropDownList tools

### ISSUE THREAD

**Issue by mbechev** [telerik/kendo-angular#4626]
### Description

The focus is trapped from the DropDownList tools when the focus is directly returned to the tool which prevents navigating to any other tool with arrow keys. 

### Steps To Reproduce

1. Use Shift + Tab to navigate from Editor content to Toolbar.
2. Navigate to any DropDownList and select any option from it.
3. Use `Tab` to focus back the content area.
4. Repeat step 1 which will focus the last used DropDownList tool. Arrow keys no longer focus the previous or next tool.

https://stackblitz.com/edit/angular-ne2vzqyc

### Screenshots or video

https://github.com/user-attachments/assets/4eec6f84-655f-419f-920b-97e418bb0d97

### Actual Behavior

The Arrow keys don't work when DropDownList tool is directly focused with Shift+Tab.

Possibly related to https://github.com/telerik/kendo-angular/issues/4288.

### Expected Behavior

The user should be able to navigate to any other toolbar tool, using the arrow keys, regardless of the initially focused tool.

### Browser

Chrome

### Browser version

latest

### OS type

MacOS
